EventFlow.Redis/EventStore/IEventStreamCollectionResolver.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace EventFlow.Redis.EventStore;
+
+public interface IEventStreamCollectionResolver
+{
+    /// <summary>
+    /// Returns the ids of all streams (aggregates) used by eventflow.
+    /// </summary>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    Task<IEnumerable<PrefixedKey>> GetStreamIdsAsync(CancellationToken cancellationToken = default);
```

Comment:
Change type to `Task<IReadOnlyCollection<PrefixedKey>>` as returning a `IEnumerable<>` from an async method doesn't really make sense. If we wanted an enumerable type from async, it should be `IAsyncEnumerable`